sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/transforms/windowing/Window.java

```diff
@@ -599,6 +600,33 @@ public void validate(PCollection<T> input) {
       return output.setWindowingStrategyInternal(outputStrategy);
     }
 
+    @Override
+    public void populateDisplayData(DisplayData.Builder builder) {
+      builder
+          .add("windowFn", windowFn.getClass())
+          .include(windowFn);
+
+      if (trigger != null) {
+        builder.add("trigger", trigger.toString());
```

Comment:
@kennknowles 

For the trigger expression:

    AfterWatermark.pastEndOfWindow()
        .withLateFirings(AfterProcessingTime
            .pastFirstElementInPane()
            .plusDelayOf(Duration.standardMinutes(10)));

The `.toString()` is:

> AfterWatermark.AfterWatermarkEarlyAndLate(AfterWatermark.NeverTrigger, AfterProcessingTime.pastFirstElementInPane([com.google.cloud.dataflow.sdk.transforms.windowing.AfterDelayFromFirstElement$DelayFn@927df]))

Looks like we need some better handling of this.